src/OpenTelemetry.AutoInstrumentation.Native/Resource.rc

```diff
@@ -104,6 +104,9 @@ NETFRAMEWORK_MANAGED_ENTRYPOINT_SYMBOLS SYMBOLS                 "..\\bin\\Profil
 
 
 
+
```

Comment:
Even more empty lines? :- )

Reply:
I mostly copied/pasted so that there was no diff between upstream repo and this repo. I can delete these extra lines if that would be preferred 😄

Reply:
Lets keep it as it is.